.github/workflows/functional-tests.yml

```
@@ -58,7 +66,8 @@ jobs:
           sudo sed -i -e 's/listen.owner = www-data/listen.owner = runner/' /etc/php/${{ env.USE_PHP_V }}/fpm/pool.d/www.conf
           sudo systemctl restart php${{ env.USE_PHP_V }}-fpm.service
       - name: Install Backdrop
-        run: core/scripts/install.sh --db-url=mysql://root:root@localhost/backdrop
-
+        run: |
+          cp modules/${{ env.REPO_NAME }}/.github/misc/settings.local.php .
+          core/scripts/install.sh --db-url=mysql://root:root@127.0.0.1/backdrop
       - name: Run functional tests
-        run: core/scripts/run-tests.sh --force --directory=modules/${{ env.REPO_NAME }} --verbose --concurrency=2
+        run: core/scripts/run-tests.sh --force --directory=modules/${{ env.REPO_NAME }} --verbose --concurrency=color --concurrency=3 --cache 2>&1
```

```suggestion
        run: core/scripts/run-tests.sh --force --directory=modules/${{ env.REPO_NAME }} --verbose --color --concurrency=3 --cache 2>&1
```

Corrected per Zulip PM.